integrated_channels/xapi/statements/base.py

```diff
@@ -15,36 +15,27 @@ class EnterpriseStatement(Statement):
     Base statement for enterprise events.
     """
 
-    def get_actor(self, username, email):
+    def get_actor(self, user):
         """
         Get actor for the statement.
         """
         return Agent(
-            name=username,
-            mbox='mailto:{email}'.format(email=email),
+            name=u'{id}-{username}'.format(id=user.id, username=user.username),
```

Comment:
I am waiting for feedback from SumTotal/Rustici regarding this change -- it may be preferable to instead populate this field with the SSO UID value.